test/vector_search_validator/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+## Copyright 2025-present ScyllaDB
+# SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
+
+[workspace]
+members = ["crates/*"]
+default-members = ["crates/validator"]
+resolver = "3"
+
+[workspace.package]
+version = "0.1.0"
+edition = "2024"
+
+[workspace.dependencies]
+anyhow = "1.0.97"
+futures = "0.3.31"
+scylla = { version = "1.2.0", features = ["time-03"] }
+tokio = { version = "1.44.1", features = ["full"] }
+tracing = "0.1.41"
+uuid = "1.16.0"
+vector-search-validator-engine = { git = "https://github.com/scylladb/vector-store.git", rev = "3ee46a5" }
+vector-search-validator-tests = { git = "https://github.com/scylladb/vector-store.git", rev = "3ee46a5" }
```

Comment:
How does this "rev" thing work? You'll need to periodically update this number (in three separate places...) to run the right tests?